calico-policies/private-network-isolation/calico-v3/au-syd/allow-private-services-pods.yaml

```diff
@@ -70,6 +70,21 @@ spec:
       - 514
     protocol: UDP
     source: {}
+  - action: Allow
+    destination:
+      nets:
+      # IBM ATracker API : AU Syd
```

Comment:
How likely is it that these IPs will change over time, and is there a plan to keep these policies updated?

Reply:
IPs are fixed.